src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
Magic numbers should be avoided as they can make the code less readable and more difficult to maintain. In this case, '100' is a magic number. It's better to declare it as a named constant at the top of the class to give it a meaningful name and to make the code more maintainable.